app/components/l-two-columns.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div :class="b()">
+    <div :class="b('wrapper')">
+      <c-header/>
+      <div :class="b('content')">
+        <div :class="b('left')">
+          <slot name="left"></slot>
+        </div>
+        <div :class="b('main')">
+          <slot name="main"></slot>
+        </div>
+      </div>
+      <c-footer/>
+    </div>
+  </div>
+</template>
+
+<script>
+  import cHeader from '@/components/c-header';
+  import cFooter from '@/components/c-footer';
+
+  export default {
+    name: 'l-two-columns',
+    components: {
+      cHeader,
+      cFooter,
+    },
+    // components: {},
+    // mixins: [],
+
+    // props: {},
+    // data() {
+    //   return {};
+    // },
+
+    // computed: {},
+    // watch: {},
+
+    // beforeCreate() {},
+    // created() {},
+    // beforeMount() {},
+    mounted() {
+      const resizeHeader = function() {
+        const height = document.getElementsByClassName('c-header')[0].offsetHeight;
+
+        document.getElementsByClassName('l-two-columns__content')[0].style.marginTop = `${height}px`;
```

Comment:
- [ ] Same function like in the l-default component => maybe you can exclude this to a helper function?

Reply:
This function might be altered in the future, I would leave it like this atm.